numpy/core/src/umath/ufunc_object.c

```diff
@@ -4952,6 +4947,7 @@ ufunc_at(PyUFuncObject *ufunc, PyObject *args)
         return override;
     }
 
+    NPY_BEGIN_THREADS_DEF;
```

Comment:
This is a definition, so has to be before any statements (yay, pre-C99...)

Reply:
Oops - now corrected. Possibly makes more sense if the real implementation is in a new function, but I'll leave that for another PR (I'm again looking at moving the implementation of ndarray ufunc to `ndarray.__array_ufunc__`, or at least making the call there faster).